driver-core/src/main/java/com/datastax/driver/core/Connection.java

```diff
@@ -704,9 +660,7 @@ public void shutdown() {
             // we're not on an I/O thread or anything, so just call await.
             allChannels.close().awaitUninterruptibly();
 
-            // This will shut down I/O threads. Since this is called only once all connection have
-            // been individually closed, it's fine.
-            eventLoopGroup.shutdownGracefully().syncUninterruptibly();
+            nettyCustomizer.shutdownEventLoopGroup(eventLoopGroup);
```

Comment:
Maybe this should be renamed?  Depending on if the EventLoopGroup is being used for other things, the user may not want to close it.  Maybe name it "complete" "doneWithCluster", or something to indicate that the Cluster will no longer need the resources (EventLoopGroup / ByteBuf / Anything that was created to support adding to pipeline / etc.).